sklearn/linear_model/coordinate_descent.py

```diff
@@ -502,6 +503,23 @@ def enet_path(X, y, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
         return alphas, coefs, dual_gaps, n_iters
     return alphas, coefs, dual_gaps
 
+  #takes out remaining values in **params, if there are any values left
```

Comment:
Why are you doing things after the end of the function?? These checks can be made before `for i, alpha in enumerate(alphas):` above.